src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketPostBuildStatus.java

```diff
@@ -39,10 +37,6 @@ public void decorateCheckoutCommand(GitSCM scm, Run<?, ?> run, GitClient git, Ta
             return;
         }
 
-        if (!hasBitbucketScmOrBitbucketScmSource(run)) {
```

Comment:
I don't know why we had this check here? As far as I can tell, we have the serverId from when this object was constructed so why do we need to check again here?

Reply:
I can't think of why either